backend/server.js

```diff
@@ -363,6 +474,9 @@ sbp('okTurtles.data/set', PUBSUB_INSTANCE, createServer(hapi.listener, {
       })
     }))
   }
+  sbp('chelonia.persistentActions/load').catch(e => {
+    console.error(e, 'Error loading persistent actions')
+  })
```

Comment:
I think if the server fails to load these perhaps we should shut down with this error instead of proceeding so that the sysadmin knows something is critically wrong.

Reply:
Hmm, possibly. Right now, we're only using persistent actions for contract deletion, which I'd not necessarily consider 'critical' (in the sense that it's possible for the server to operate normally without this). We could shut the server down, though.

I think that if we go the shutdown route, some other changes may be needed to do that cleanly (currently, the server would start, possibly start serving clients and _then_ shutdown).

Reply:
and maybe we could shut down by default, and allow the server to run without shutting down if a flag like `--ignore-errors` is passed in?